### ISSUE TITLE
kairos 2 - edgevpn failed starting

### ISSUE THREAD

**Issue by vincentserpoul** [kairos-io/kairos#1363]
**Kairos version:**

```
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.17.3
PRETTY_NAME="Alpine Linux v3.17"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
KAIROS_NAME="kairos-core-alpine-arm-rpi"
KAIROS_VERSION="v2.0.2"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-core-alpine-arm-rpi"
KAIROS_VERSION_ID="v2.0.2"
KAIROS_PRETTY_NAME="kairos-core-alpine-arm-rpi v2.0.2"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
KAIROS_HOME_URL="https://github.com/kairos-io/kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/core-alpine-arm-rpi"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/kairos"
KAIROS_VARIANT="core"
KAIROS_FLAVOR="alpine-arm-rpi"KAIROS_NAME="kairos-alpine-arm-rpi"
KAIROS_VERSION="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-alpine-arm-rpi"
KAIROS_VERSION_ID="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_PRETTY_NAME="kairos-alpine-arm-rpi v2.0.2-k3sv1.26.4+k3s1"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues/new/choose"
KAIROS_HOME_URL="https://github.com/kairos-io/provider-kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/kairos-alpine-arm-rpi"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/provider-kairos"
KAIROS_VARIANT="core"
```

**CPU architecture, OS, and Version:**

`Linux xx 5.14.21-150400.24.33-default #1 SMP PREEMPT_DYNAMIC Fri Nov 4 13:55:06 UTC 2022 (76cfe60) aarch64 GNU/Linux
`
**Describe the bug**

if seems edgevpn is not supplied the right option. I tried also deactivate it, but it seems the p2p option does not work.

` * Starting edgevpn ...edgevpn: unrecognized option: enable-healthchecks`

**To Reproduce**

used config

```
#cloud-config

hostname: homelab-{{ trunc 4 .MachineID }}

users:
  - name: kairos
    ssh_authorized_keys:
      - github:xxx

k3s:
  enabled: true
  args:
    - --disable=traefik,servicelb

kubevip:
  eip: "192.168.2.100"
  enable: true

p2p:
  disable_dht: true #Enabled by default
  network_token: 
```

**Expected behavior**

**Logs**
/var/log/kairos/agent.log

```
Provider kairos at /system/providers/agent-provider-kairos had an error: Failed setup VPN: could not start svc: failed starting service: edgevpn.  * Caching service dependencies ... [ ok ]
 * Starting edgevpn ...edgevpn: unrecognized option: enable-healthchecks
Usage: supervise-daemon [options]

Options: [ A:a:D:d:e:g:H:I:Kk:m:N:p:R:r:s:Su:1:2:3ChqVv ]
  -a, --healthcheck-timer <arg>     set an initial health check delay
  -A, --healthcheck-delay <arg>     set a health check timer
      --capabilities <arg>          Set the inheritable, ambient and bounding capabilities
      --secbits <arg>               Set the security-bits for the program
      --no-new-privs                Set the No New Privs flag for the program
  -D, --respawn-delay <arg>         Set a respawn delay
  -d, --chdir <arg>                 Change the PWD
  -e, --env <arg>                   Set an environment string
  -g, --group <arg>                 Change the process group
  -I, --ionice <arg>                Set an ionice class:data when starting
  -K, --stop                        Stop daemon
  -k, --umask <arg>                 Set the umask for the daemon
  -m, --respawn-max <arg>           set maximum number of respawn attempts
  -N, --nicelevel <arg>             Set a nicelevel when starting
      --oom-score-adj <arg>         Set OOM score adjustment when starting
  -p, --pidfile <arg>               Match pid found in this file
  -P, --respawn-period <arg>        Set respawn time period
  -R, --retry <arg>                 Retry schedule to use when stopping
  -r, --chroot <arg>                Chroot to this directory
  -s, --signal <arg>                Send a signal to the daemon
  -S, --start                       Start daemon
  -u, --user <arg>                  Change the process user
  -1, --stdout <arg>                Redirect stdout to file
  -2, --stderr <arg>                Redirect stderr to file
  -3, --reexec                      reexec (used internally)
  -h, --help                        Display this help output
  -C, --nocolor                     Disable color output
  -V, --version                     Display software version
  -v, --verbose                     Run verbosely
  -q, --quiet                       Run quietly (repeat to suppress errors)

 * failed to start edgevpn
                                               [ !! ]
 * ERROR: edgevpn failed to start
 (exit status 1)
```

**Additional context**

**Comment by mauromorales** [comment]
I check the installed version of edgevpn on arm generic and the option seems to be there

```
:/ # file $(which edgevpn)
/usr/bin/edgevpn: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, stripped
:/ # edgevpn api --help | grep enable-health
   --enable-healthchecks                          [$ENABLE_HEALTHCHECKS]
```

will have to test on the RPi

**Comment by vincentserpoul** [comment]
I just retry with 2.0.3, I don't have the issue anymore. I've not seen anything related to it in the changelog, so I can't really say it's fixed in 2.0.3, but it seems.

**Comment by mauromorales** [comment]
Thanks for reporting @vincentserpoul ... I'll close for now and we can re-open if the issue persists
